cmd/agent/app/run.go

```diff
@@ -178,6 +179,11 @@ func StartAgent() error {
 
 	log.Infof("Starting Datadog Agent v%v", version.AgentVersion)
 
+	// init settings that can be changed at runtime
+	if err := settings.InitRuntimeSettings(); err != nil {
+		log.Warnf("Can't initiliaze the runtime settings: %v", err)
+	}
+
```

Comment:
Should we do this for dogstatsd too ?

Reply:
Dogstatsd doesn't have the `config` command so the answer is no. If we should add the `config` command (+ runtime settings support) is an interesting question.